docs/gen/ex_DIDNOTCONVERGE_msg.rst

```diff
@@ -1,7 +1,4 @@
 .. comment: generated by t_lint_didnotconverge_bad
 .. code-block::
 
-   -V{t#,#}+    Vt_lint_didnotconverge_bad___024root___change_request
-   -V{t#,#}+    Vt_lint_didnotconverge_bad___024root___change_request_1
-   -V{t#,#}        CHANGE: t/t_lint_didnotconverge_bad.v:14: a
-   %Error: t/t_lint_didnotconverge_bad.v:7: Verilated model didn't converge
+   Aborting...
```

Comment:
Didn't see this on your TODO but we need some way for user to resolve non-convergence/infinite scheduler loops.

Reply:
Dnabling debug still works, it's just not enabled automatically when we are about to fail. Need to track down why this is set to Abort rather than the Error line.

Reply:
I have added some code, which will dump the currently active triggers before the Abort (i.e.: the reason we think we need to keep going). Hopefully that is enough for now and can re-run with debug for more details?

Later could add a runtime debug option that dumps the triggers only for a less noisy experience.

Reply:
As long as you think it's enough for a user to have a reasonable chance of debugging the common case of events looping forever without advancing time (with VL_DEBUG defined but debug not turned on).

Reply:
Well, I don't think turning on debug and blasting the execution trace on screen gives you a much better chance without getting in deep in the generated C++ :/. At that point you might as well turn on debug to get more info?

The really nice way of doing it would be to save the last N trigger states and dump all on non-convergence, but I would prefer not add that right now to avoid getting a megapatch.

Reply:
BTW the current code still tells the user what "changed", in this example it's signal 'b', maybe that could be more legible with full hierarchical paths.